CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Pulumiverse
+
+:+1::tada: Thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing. These are just
+guidelines, not rules, so use your best judgment and feel free to propose
+changes to this document in a pull request.
+
+## Issues and Pull Requests
+
+- If you're not sure about adding something, open an issue to discuss it.
+- Feel free to open a Pull Request early so that a discussion can be had as
+  changes are developed.
+
+## Commit Messages and Pull Request Titles
+
+We use the
+[Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
+specification to standardize our commit history. To enforce this convention on
+commit messages and/or pull request titles, we use the
+[Semantic Pull Requests](https://github.com/probot/semantic-pull-requests) bot.
+
+The commit message summary (or pull request title) is constructed by prepending
+the type of change being made (e.g., feat, fix, refactor), followed by an
+imperative, present tense sentence (without a period). Example:
+`fix: make header bold`.
+
+If you are still working on your pull request, prepend `WIP:` to indicate that
+it's work in progress or use the draft pull requests feature.
+
+### Pull Request Title
+
+Same as commit messages, prepend the type of change being made (refactor, fix,
+chore, feat, etc.) Example: `docs: add linux setup instructions`
+
+## Need Help?
+
+If any of this information confusing, incorrect, or incomplete, feel free to
+join the [Pulumi Community slack][pulumi-slack] for help.
```

Comment:
Is there a particular channel pulumiverse we could point contributors to? Should there be?

Reply:
Yes, thank you. There's probably also a link we should add :D

Reply:
@cobraz can you mention the `#pulumiverse` channel in this section?

Reply:
Also, the link should be updated to this: https://slack.pulumi.com/